### DISCUSSION TITLE
website redirects but won't open

### DISCUSSION THREAD

**Discussion by swhit434** [community/community#36254] [Other Feature Feedback, Questions, & Ideas]
Hello, when I type in my gitub address, it redirects to my website through namecheap but will not open. how do I fix this?

**Comment by jfullstackdev** [comment]
not very sure about that, this is your site on my end 

![image](https://user-images.githubusercontent.com/47092464/196041435-013be76c-d55a-42f2-b7b2-ccf78a115340.png)

do you have any proof, that domain belongs to you ? or can you show the custom domain setup ?

**Comment by ossanna16** [comment]
Hey there @swhit434 and thank you for asking a good question! I'm glad you found our community 🙂
